modules/users/server/controllers/users.authentication.server.controller.js

```diff
@@ -72,6 +73,10 @@ exports.signup = function (req, res) {
 
       user.emailToken = authenticationService.generateEmailToken(user, salt);
 
+      if (req.body.acquisitionStory != null) {
```

Comment:
This will never end up being `null` if it's sent from the form: it'll likely be an empty string.

How about just `if (!req.body.acquisitionStory)` which would also ignore empty texts as well missing body field?